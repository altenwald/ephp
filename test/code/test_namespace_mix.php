<?php

namespace Project\Codes;

const OK = 1;
const FAIL = 0;

class Error {
    function __construct() {
        print __NAMESPACE__ . "\n";
    }
}

function errno($n) {
    print __NAMESPACE__ . " | gave => $n\n";
}

namespace Project {
    print "give me an OK => " . Codes\OK . "\n";
    print "give me a FAIL => " . Codes\FAIL . "\n";

    $e = new Codes\Error();
    Codes\errno(Codes\OK);
    Codes\errno(Codes\FAIL);
}
