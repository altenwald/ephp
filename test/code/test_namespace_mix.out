
Fatal error: Cannot mix bracketed namespace declarations with unbracketed namespace declarations in {{CWD}}/test/code/test_namespace_mix.php on line 18
